README.md

```diff
@@ -109,7 +114,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+see q2.3
```

Comment:
Nice shade of green :)

You can simplify that diagram by merging states 1 and 2:

![image](https://user-images.githubusercontent.com/10648/78700769-80092a80-78cb-11ea-8dce-e50069b74084.png)

Otherwise perfect

# -2